source/state/vault/index.ts

```diff
@@ -1,6 +1,7 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { ethers } from 'ethers';
 import loadsh from 'lodash';
+import lodash from 'lodash';
```

Comment:
There seem to be two repeated imports from lodash here

Reply:
fixed